src/content/docs/query-your-data/nrql-new-relic-query-language/get-started/nrql-syntax-clauses-functions.mdx

```diff
@@ -1812,7 +1812,7 @@ SELECT histogram(duration, 10, 20) FROM PageView SINCE 1 week ago
 
       <CollapserGroup>
         <Collapser title={<><InlineCode>capture()</InlineCode> within a <InlineCode>SELECT</InlineCode> clause condition</>}>
-          The following will select the domain name of the website, removing https&#x3A;// and any paths following the .com
+          The following will select the domain name of the website, removing `https://` and any paths following the `.com`
```

Comment:
@austin-schaefer is the "r'" correct before "https"?

Reply:
I'm not sure, I didn't touch that part of the doc. Looking at the surrounding examples though, it looks like it's part of the regex syntax. The intro to that section says:

> Regex expressions in NRQL use Python-like syntax, `r'...'`.